Introductory/collaborationtypes/EngineMonitor.json

```diff
@@ -5,13 +5,8 @@
     "MergeStreams" : {
       "configuration" : {
         "childStreams" : [ "UpdateSystemStatus" ],
-        "outboundResource" : null,
```

Comment:
The UI no longer exports unused properties (which is good), which is why a bunch of lines were removed from the old export.